quiche/src/recovery/congestion/bbr/mod.rs

```diff
@@ -337,6 +349,10 @@ fn has_custom_pacing() -> bool {
     true
 }
 
+fn state_str(r: &Congestion, _now: Instant) -> &'static str {
```

Comment:
CI is complaining about "method `state_str` is never used", so I wonder if this function needs to go behind `#[cfg(feature = "qlog")]`